about.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Averi Kitsch</title>
+  <link rel="stylesheet" href="stylesheets/normalize.css" />
+  <link rel="stylesheet" href="stylesheets/main.css" />
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Fira+Sans+Extra+Condensed">
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Mrs+Sheppards">
+
+</head>
+  <body>
+    <div id="wrapper">
+
+        <header id="header">
+          <span id="nav-left"><a href="index.html">ak</a></span>
+          <div id="nav-right">
+            <ul>
+              <li><a href="index.html">home</a></li>
+              <li><a href="portfolio.html">portfolio</a></li>
+              <li><a href="blog.html">blog</a></li>
+              <li id="select"><a href="about.html">about</a></li>
+            </ul>
+          </div>
+        </header>
+
+        <article id="about-me">
+          <section class="about-left">
+            <h2>ABOUT ME</h2>
+            <p>
+              I am currently a full-stack software development student at Ada Developers Academy.
+            </p>
+            <p>
+              I have worked in the medical imaging field for 6 years and have gained experience both utilizing and developing imaging processing pipelines. My work aims to develop tools to both expand basic biological knowledge through biomedical image data and to address practical application in real clinical problems.
+            </p>
+
+
+          </section>
+          <section class="about-right">
+            <h2>WHAT I KNOW</h2>
+            <p class="lang">ruby</p>
+            <p class="lang">matlab</p>
+            <p class="lang">html&css</p>
```

Comment:
just an interesting thing to note: you might want to put this ampersand as `&amp;`. Not required. Read more here https://developer.mozilla.org/en-US/docs/Glossary/Entity